webapp/crashstats/settings/base.py

```diff
@@ -65,6 +65,7 @@ def path(*dirs):
     ),
 )
 
+
```

Comment:
was this on purpose? if so that's fine, just looked out of place and want to make sure

Reply:
Whoops! I added something to this file and then changed how I did everything and removed it, but missed an empty line. I'll fix that.